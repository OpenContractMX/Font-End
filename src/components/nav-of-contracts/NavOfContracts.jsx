import React from "react";
import { Link } from "react-router-dom";

import "./NavOfContracts.scss";

export const NavOfContracts = () => {
  return (
    <nav className="nav-contracts">
      <Link to="/statistic">
        <button type="button" className="nav-contracts--btn-volver">
          <i class="fas fa-arrow-left icon-volver"></i>
          Volver
        </button>
      </Link>

      <button type="button" className="nav-contracts--ver-todos">
        <span>Ver todos</span>
      </button>
    </nav>
  );
};
