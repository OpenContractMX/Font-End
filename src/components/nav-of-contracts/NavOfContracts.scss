@import "../../assets/scss/reset";
@import "../../assets/scss/variables";
@import "../../assets/scss/mixins";

.nav-contracts {
  width: 100%;
  // display: flex;
  // justify-content: space-between;
  margin: 24px 0 24px 0;

  &--btn-volver,
  &--ver-todos {
    width: 108px;
    height: 36px;
    border: $color-dark-primary;
    border-radius: 8px;
    cursor: pointer;
    color: $color-light-primary;
    background-color: $color-dark-primary;
    @include subTittleMobile;

    & i {
      margin-right: 8px;
    }
  }
  &--btn-volver {
    position: fixed;
  }
  &--ver-todos {
    background-color: $color-light-primary;
    color: $color-dark-primary;
  }
}
