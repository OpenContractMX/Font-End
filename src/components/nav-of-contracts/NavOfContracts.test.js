import React from "react";
import { BrowserRouter } from "react-router-dom";

import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { NavOfContracts } from "./NavOfContracts";

it("[render content `statistic / NavOfContracts`]", () => {
  const { asFragment } = render(
    <BrowserRouter>
      <NavOfContracts />
    </BrowserRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
