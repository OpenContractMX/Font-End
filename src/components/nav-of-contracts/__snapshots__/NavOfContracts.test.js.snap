// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[render content \`statistic / NavOfContracts\`] 1`] = `
<DocumentFragment>
  <nav
    class="nav-contracts"
  >
    <a
      href="/statistic"
    >
      <button
        class="nav-contracts--btn-volver"
        type="button"
      >
        <i
          class="fas fa-arrow-left icon-volver"
        />
        Volver
      </button>
    </a>
  </nav>
</DocumentFragment>
`;
